deploy.sh

```diff
@@ -1,25 +1,25 @@
 #!/bin/sh
 
 if [[ $# -eq 0 ]] ; then
-    echo 'Please specify the services to deploy. Services available: "rns" "storage".'
+    echo 'Please specify the services to deploy. Services available: "rns" "storage" "triggers".'
```

Comment:
I would rename the whole service and folder to `notifications` just to be consistent with UI and how we name the services in the other parts of the Marketplace.

Reply:
Done